roles/hosts-file/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+  - name: logic flags | compute
+    set_fact:
+        do_clear: >
+            {{ hosts_file_clear_ansible_group != "" }}
+
+  - name: hosts file | probe
+    command: cat /etc/hosts
+    register: hosts_probe
+    when: "{{ do_clear|bool }}"
+
+  - name: hosts file | filter
+    set_fact:
+        filtered_hosts: >
+            {{ hosts_probe.stdout|hosts_file_filter(
+                groups[hosts_file_clear_ansible_group],
+                domain,
+                hostvars,
+                extra_clear_entries,
+                ansible_hostname,
+                ansible_nodename,
+                ansible_fqdn) }}
+    when: "{{ do_clear|bool }}"
+
+  - name: hosts file | clear
+    copy:
+        content: "{{ filtered_hosts }}"
+        dest: /etc/hosts
+    when: "{{ do_clear|bool }}"
+
+  - name: hosts file | amend
+    lineinfile:
+        dest: /etc/hosts
+        regexp: "^{{ item[0]|regex_escape() }}"
+        line: >
+            {{ item[0] }} {{ item[1]|join(" ") }}
+        state: >
+            {{ "present" if item[0] in ("127.0.0.1", "::1") else state }}
+    with_items: >
+        {{  groups[hosts_file_ansible_group]
+          | hosts_file_entries(hosts_file_net_interface,
+                               domain,
+                               hostvars,
+                               extra_entries,
+                               local_entries|bool,
+                               ansible_hostname,
+                               ansible_nodename,
+                               ansible_fqdn) }}
+
```

Comment:
I am receiving the following error:
```
TASK: [hosts-file | hosts file | amend] *************************************** 
fatal: [name] => Failed to template {{  groups[hosts_file_ansible_group]
  | hosts_file_entries(hosts_file_net_interface,
                       domain,
                       hostvars,
                       extra_entries,
                       local_entries|bool,
                       ansible_hostname,
                       ansible_nodename,
                       ansible_fqdn) }}
: template error while templating string: no filter named 'hosts_file_entries'
fatal: [data-01] => Failed to template {{  groups[hosts_file_ansible_group]
  | hosts_file_entries(hosts_file_net_interface,
                       domain,
                       hostvars,
                       extra_entries,
                       local_entries|bool,
                       ansible_hostname,
                       ansible_nodename,
                       ansible_fqdn) }}
: template error while templating string: no filter named 'hosts_file_entries'

FATAL: all hosts have already failed -- aborting
```

Running this down a little bit it looks like [hosts_file_entries](https://github.com/opadron/gobig/blob/hosts-file-role/filter_plugins/gobig.py#L12-L20) is missing the host_list argument?